packages/ra-auth-cognito/Readme.md

```diff
@@ -96,6 +101,71 @@ To support react-admin [identity feature](https://marmelab.com/react-admin/AuthP
 
 This `authProvider.getPermissions` method returns an array of [the groups assigned to the user](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-user-groups.html?icmpid=docs_cognito_console_help_panel).
 
+## `<LoginForm>`
+
+A component that renders a login form. It handles first login with temporary passwords. Use it if you just want to customize the login page design:
+
+```tsx
+import { Box, Card, CardContent, CardMedia, CssBaseline } from '@mui/material';
+import { LoginForm } from 'ra-auth-cognito';
+
+export const MyLoginPage = () => {
+    return (
+		<>
+			<CssBaseline />
+			<Box>
+                <Card>
+                    <CardMedia
+                        sx={{ height: 140 }}
+                        image="/login_background.jpg"
+                    />
+                    <CardContent>
+                        <LoginForm redirectTo="/" />
+                    </CardContent>
+				</Card>
+			</Box>
+		</>
```

Comment:
strange indentation

Reply:
I just opened this file on my IDE and the indentation looks correct. I think the weird indentation comes from the Github diff viewer

Reply:
right, I noticed the same